spec/features/courses/sites/edit_spec.rb

```diff
@@ -113,6 +113,7 @@
 
       locations_page.save.click
 
+      expect(page.title).to have_content('Error:')
```

Comment:
```suggestion
      expect(course_page.title).to have_content('Error:')
```

Reply:
The `.title` is already being used for the `h1` on the page.

I don't think it's wrong in this situation to use the `page.title` which already selects the correct element vs having to update all our page objects with a new `title` selector that is actually the `css('head title')`

Reply:
oh fair enough